### DISCUSSION TITLE
"prepare for an expedition" order execution is obsolete and must be renewed.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#12911] [Ideas and suggestions]
When fulfilling the order “prepare for an expedition, the bot tries to add 5 oxygen tanks to his inventory. In the previous version of the game this was correct. But now when the tanks do not stacks in inventory, this is a problem. Moreover, 5 tanks it is 5000 seconds of breathing, that is, an hour and a half.
Please, make the bots takes 2 tanks instead of 5.
Second problem: Bots do not check if they already have enough tanks in their belt or backpack.
The belt is fireproof, so I always keep the tanks in the belt. 
But the bot takes 5 tanks, filling up half of the inventory, even if their belt already filled with tanks.

**Comment by Regalis11** [comment]
This will be fixed in the next hotfix. :) https://github.com/Regalis11/Barotrauma/issues/12878

**Comment by Hazmat616** [comment]
BELTS ARE FIREPROOF?

**Reply by pavel4win** [reply]
Wiki say so. When you struggle with fire, tanks in belt do not deteriorate

https://barotraumagame.com/wiki/Containers
